apps/explorer/src/index.css

```diff
@@ -14,7 +14,132 @@
     --steel-dark: theme('colors.steel.dark');
 }
 
-body {
-    @apply antialiased;
-    @apply font-sans text-offblack;
+@layer base {
+    body {
+        @apply antialiased;
+        @apply font-sans text-offblack;
+    }
+}
+
+@layer components {
+    .home-page-grid-container {
+        @apply grid grid-cols-1 gap-4;
+    }
+
+    @screen sm {
+        .home-page-grid-container {
+            @apply grid-cols-[200px,auto];
+            grid-template-areas:
+                'tps         tps'
+                'network     network'
+                'epoch       gas-price'
+                'checkpoint  gas-price'
+                'node-map    node-map'
+                'activity    activity'
+                'validators  validators'
+                'packages    packages';
+        }
+
+        .home-page-grid-container-with-sui-token {
+            grid-template-areas:
+                'tps         sui-token'
+                'network     network'
+                'epoch       gas-price'
+                'checkpoint  gas-price'
+                'node-map    node-map'
+                'activity    activity'
+                'validators  validators'
+                'packages    packages';
+        }
+
+        .grid-area-tps {
```

Comment:
I probably would have defined these inline with `style`, and then just added a `grid-template-areas` for the base case where things are just defined top-down. IMO that's slightly nicer here because it cuts down on one-line class definitions and the base case is no longer special and still uses the grid template areas..